flieLogger/index.js

```diff
@@ -0,0 +1,73 @@
+/*
+ *  *******************************************************************************
+ *  * Copyright (c) 2020 Edgeworx, Inc.
+ *  *
+ *  * This program and the accompanying materials are made available under the
+ *  * terms of the Eclipse Public License v. 2.0 which is available at
+ *  * http://www.eclipse.org/legal/epl-2.0
+ *  *
+ *  * SPDX-License-Identifier: EPL-2.0
+ *  *******************************************************************************
+ *
+ */
+const pino = require('pino')
+const path = require('path')
+const fs = require('fs')
+
+const defaultFormat = {
+  level: 'debug',
+  messageKey: 'message',
+  timestamp: () => {
+    return ', "timestamp":' + timeStamp()
+  },
+  formatters: {
+    level: (level) => ({ level })
+  }
+}
+
+const timeStamp = () => {
+  return new Date(Date.now()).toISOString()
+}
+
+const dirName = '/var/log/iofog-microservices/'
```

Comment:
dirname should be configurable, with a default value

Reply:
Updated with dirName configurable